docs/_pages/docs/getting_started/getting-started.md

```diff
@@ -89,12 +89,18 @@ The `regolith watch [profile-name]` command works the same as `regolith run`, bu
 it will watch your source files and rerun the profile when they change. If you're
 using `regolith run` you have to do it manually every time.
 
-A single run copies your source files into a temporary folder, runs all of the
-filters of the profile and moves the files to target location defined in the
-"export" property of the profile. By default the export is set to "development",
-which means that the files will be copied to the `development` pack folders of
-`com.mojang`. The names of folders created in this export mode are based on
-the name of the project like `project_name_bp` and `project_name_rp`.
+A single run copies your source files into a temporary folder, then runs all
```

Comment:
This paragraph could be rewritten to be a list, like 1) 2) 3) (bullet points) to make it easier to see the order.

Reply:
Done. Except that this text was moved to the `filter-run-modes.md`.